pyproject.toml

```diff
@@ -12,14 +12,14 @@ packages = [{include = "poetry_polylith_plugin"}]
 poetry-polylith-plugin = "poetry_polylith_plugin:PolylithPlugin"
 
 [tool.poetry.dependencies]
-python = "^3.7"
-poetry = {version = "^1.2.0a2", allow-prereleases = true}
+python = "^3.8"
```

Comment:
If I remember this one correctly, I aligned with the same version as the Poetry tool expects (currently v 3.7). But maybe that is not required.